src/lib/Modal.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+
+  export let title: string;
+  export let open = false;
+</script>
+
+<!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
The outer `background` div is fullscreen and adds the fade. If you click it, we close the modal. There's the a11y warning because it's not a `<button>`. I think that's actually fine from an a11y perspective; there's an explicit close button inside the modal. If you were tabbing over everything and selected the entire screen, it'd be confusing as a button anyway